### ISSUE TITLE
lpr_max_len

### ISSUE THREAD

**Issue by nightfuryyy** [sirius-ai/LPRNet_Pytorch#9]
"lpr_max_len" in LPRNet.py. What does it relate to the model? thank you so much for awsome repo.

**Comment by sirius-ai** [comment]
lpr_max_len is the actual license plate max lenth, it should be less or equal than the {'T'/2} of model output logits.

**Comment by nightfuryyy** [comment]
@sirius-ai thank you

**Comment by jeungmin717** [comment]
in train_LPRNet.py

there is  code
T_length = 18 # args.lpr_max_len
but ,
args.lpr_max_len is 8 by default


T_length is why  18? 
if T_length and args.lpr_max_len is the same value then shouldn't both value be 8 by default?

**Comment by jeungmin717** [comment]
@nightfuryyy , @sirius-ai 
in code train_LPRNet.py

there is 
T_length = 18 # args.lpr_max_len
but ,
args.lpr_max_len is 8 by default

T_length is why  18? 
if T_length and args.lpr_max_len is the same value then shouldn't both value be 8 by default?
Thank you so much :)

**Comment by KunxiSun** [comment]
> @nightfuryyy , @sirius-ai in code train_LPRNet.py
> 
> there is T_length = 18 # args.lpr_max_len but , args.lpr_max_len is 8 by default
> 
> T_length is why 18? if T_length and args.lpr_max_len is the same value then shouldn't both value be 8 by default? Thank you so much :)

In my opinion, T_length is a hyperparameter that can be set to any number. Here the author sets it to 18, which limits the maximum length of the license plate to 18. The difference between it and lr_max_length is that the lr_max_length is the actual maximum length, while during the actual training process, the license plate will can have a empty letter, so there are blank place left in the T_length.
